relayer-cli/src/commands/clear.rs

```diff
@@ -0,0 +1,90 @@
+use abscissa_core::clap::Parser;
+use abscissa_core::{Command, Runnable};
+
+use ibc::core::ics24_host::identifier::{ChainId, ChannelId, PortId};
+use ibc::events::IbcEvent;
+use ibc_relayer::link::error::LinkError;
+use ibc_relayer::link::{Link, LinkParameters};
+
+use crate::application::app_config;
+use crate::cli_utils::ChainHandlePair;
+use crate::conclude::Output;
+use crate::error::Error;
+
+/// `clear` subcommands
+#[derive(Command, Debug, Parser, Runnable)]
+pub enum ClearCmds {
+    /// Clear outstanding packets (i.e., packet-recv and packet-ack)
+    /// on a given channel in both directions.
+    Packets(ClearPacketsCmd),
+}
+
+#[derive(Debug, Parser)]
+pub struct ClearPacketsCmd {
+    #[clap(required = true, help = "identifier of the destination chain")]
+    dst_chain_id: ChainId,
+
+    #[clap(required = true, help = "identifier of the source chain")]
+    src_chain_id: ChainId,
+
+    #[clap(required = true, help = "identifier of the source port")]
+    src_port_id: PortId,
+
+    #[clap(required = true, help = "identifier of the source channel")]
+    src_channel_id: ChannelId,
+}
```

Comment:
Since the goal is to clean both directions there shouldn't be a ``src` and `dst`. Maybe `a` and `b` but i think we can just specify the ID-s on one chain.
```suggestion
pub struct ClearPacketsCmd {
    #[clap(required = true, help = "chain identifier")]
    chain_id: ChainId,

    #[clap(required = true, help = "identifier of the port")]
    port_id: PortId,

    #[clap(required = true, help = "identifier of the channel")]
    channel_id: ChannelId,
}
```

Reply:
We still need to identify the other chain. What would be a good name for the flag currently named `dst_chain_id`?
These are positional parameters, so these names do not factor in the CLI, but I agree that the names and descriptions should be accurate.

Reply:
Or, is there a way to determine all counterparty IDs from the channel identified on "the chain"? The `Link` API does not seem to provide that; it also requires the type of the destination chain to be instantiated.

Reply:
There are other commands that do this, for example `QueryUnreceivedAcknowledgementCmd`.

Reply:
@ancazamfir With 248940dece0c7412ec221e557cc3232deff9c30c, the usage looks like this:

```
hermes clear packets <COUNTERPARTY_CHAIN_ID> <CHAIN_ID> <PORT_ID> <CHANNEL_ID>
```

Reply:
can you check 98b2a5cfdb924ce6d26f7097566eb6c50a2a6865 where i removed the counterparty chain parameter?

Reply:
Thanks @ancazamfir, it works with the three parameters. I have adjusted the descriptions for the parameters and the command to follow this change.